README.md

```diff
@@ -11,7 +11,13 @@ You can find some basic setup instructions and links to the RuboCop OSS project
 ### Installation
 
 1. If you haven't already, [install the Code Climate CLI](https://github.com/codeclimate/codeclimate).
-2. Run `codeclimate engines:enable rubocop`. This command both installs the engine and enables it in your `.codeclimate.yml` file.
+2. Enable the engine by adding the following under `plugins` in your `.codeclimate.yaml`:
+    ```yaml
+    plugins:
+      rubocop:
+        enabled: true
+    ```
+2. Run `codeclimate engines:install`. This command installs the engines enabled it in your `.codeclimate.yml` file.
```

Comment:
I don't think this is totally  true. If I recall, `engines:install` just pulls the docker image. We should still be fine without this step though, no?

Reply:
There's actually a typo but the meaning is it pulls the docker images.
```suggestion
2. Run `codeclimate engines:install`. This command installs the engines enabled in your `.codeclimate.yml` file.
```
I guess in order to run `analyze` one first have to run `engine:install` but I haven't tried it yet.

Reply:
If you do step #2, setting up `.codeclimate.yml`, you can do `analyze` right after.

Reply:
`engines:install` is useful to pull most recent version of the plugin if you have an old image, but not critical for the initial setup.